config/form_profile_mappings/20-0996.yml

```diff
@@ -1,12 +1,8 @@
 data:
   attributes:
     veteran:
-      addressLine1: [contact_information, address, street]
-      addressLine2: [contact_information, address, street2]
-      city: [contact_information, address, city]
-      stateOrProvinceCode: [contact_information, address, state]
-      zipPostalCode: [contact_information, address, postal_code]
-      countryCode: [contact_information, address, countryCode]
-      phoneNumber: [contact_information, us_phone]
-      emailAddress: [contact_information, email]
-      ssnLastFour: [identity_information, ssn_last_four]
+      address:
+        zipCode5: [contact_information, address, postal_code]
```

Comment:
postal_code may be international (letters/numbers) But wont' it most likely be  Zip+4 for US addresses?


the schema wants ^[0-9]{5}$

Reply:
ah! good catch. i'm not super familiar with the prefill code / vets360 fields yet